specs/flashtestations.md

```diff
@@ -403,34 +387,34 @@ The most frequent operation is checking if an address is valid for a specific wo
 function isAllowedWorkload(workloadId, address) → boolean
 ```
 
-This simply checks if the address is currently associated with the specified workload. This operation must be highly gas-efficient as it may run on every block.
+This simply checks if the address is currently associated with the specified workload and has not been marked as outdated. This operation must be highly gas-efficient as it may run on every block.
 
 #### 2. Registration
 
 When an attestation successfully passes verification:
 
 ```
-function addAddress(workloadId, tcbHash, address, quote)
+function addAddress(workloadId, address, quote)
 ```
 
 This operation:
 1. Records that this address has been validated for this workload
-2. Associates it with the specific endorsement bundle (`tcbHash`) that validated it
-3. Stores the raw attestation quote for future reference and verification
-3. If the address was previously registered for this workloadId with a different tcbHash, the old entry is replaced
+2. Stores the raw attestation quote for future reference and verification
+3. If the address was previously registered for this workloadId, the old entry is replaced
 
-#### 3. Endorsement Management
+#### 3. Attestation Verification
 
-When Intel endorsements become stale or insecure:
+To verify if an attestation is still valid against current endorsements:
 
 ```
-function removeEndorsement(tcbHash, endorsement)
+function verifyAttestation(address) → boolean
```

Comment:
Shouldn't this be function verifyAttestation(address, workloadId) → boolean

?

I say this because if we just pass `address` than it is ambiguous which `quote` to verify against, since there will be 1 quote for each (address, workloadId) tuple.